docs/_static/css/custom.css

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Please remove this file, only needed for Sphinx (Read the Docs).  Actually, the whole `_static` sub-directory.

Reply:
done